Program.cs

```diff
@@ -1,7 +1,30 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿using System;
+
+namespace Training {
+   internal class Program {
+      static void Main (string[] args) {
+         string[] abecedarianArr = { "glory", "floor", "ant", "victory", "aegilops", "access" };
+         int index = 0;
+         foreach (string word in abecedarianArr) {

```

Comment:
Since you require index, you could have used for-loop instead of foreach.

Reply:
Used for-loop instead of foreach-loop.